### ISSUE TITLE
Better import handling: try to import all cards, and show to the user those which cannot be imported

### ISSUE THREAD

**Issue by Altonss** [TheLastProject/Catima#685]
I thought about a possible enhancement related to the issue #661 . Instead of just quitting the whole import, we could import the cards that are "compatible"/"readable" by the import script and just don't import the once that fails. After the import a popup could show up saying: "1 card could not be imported because of a formatting error".
But this is just an enhancement idea and not a priority at all :)

**Comment by TheLastProject** [comment]
I would like to first fix #513, then we can do this more asynchronously too. But yeah, would be nice.

**Comment by Altonss** [comment]
> I would like to first fix #513, then we can do this more asynchronously too. But yeah, would be nice.

I completely agree :) This issue is just to archive some ideas for future development.
